AlgoritmicTask1.iml

```diff
@@ -1,11 +1,86 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Idea files should be ignored by Git. What if I am using Eclipse or Netbeans?

Reply:
Oh my mistake, I forgot about .gitignore.